File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends SpecialUberCar{

    public UberBlack(String license, Account driver,Map<String, Map<String,Integer>> typeCarAccepted,ArrayList<String> seatsMaterial){
        super(license, driver, typeCarAccepted, seatsMaterial);
    }
}

File: JS/classes/uber-classes/uberBlack.js
const SpecialUberCar = require("./specialUberCar").SpecialUberCar;

class UberBlack extends SpecialUberCar{
    constructor(license,driver,typeCarAccepted,seatsMaterial){
        super(license,driver,typeCarAccepted,seatsMaterial);
    }
}

exports.UberBlack = UberBlack;